src/main/java/com/ldtteam/structures/blueprints/v1/BlueprintUtil.java

```diff
@@ -70,20 +72,17 @@ public static Blueprint createBlueprint(World world,
             short x = (short) (mutablePos.getX() - pos.getX()), y = (short) (mutablePos.getY() - pos.getY()),
                 z = (short) (mutablePos.getZ() - pos.getZ());
 
-            if (!modName.equals("minecraft"))
+            if (!modName.equals("minecraft") && !modName.equals(MOD_ID))
             {
-                if (!requiredMods.contains(modName))
-                {
-                    if (!ModList.get().getModContainerById(modName).isPresent())
-                    {
-                        requiredMods.add(modName);
-                    }
-                }
-                else if (!ModList.get().getModContainerById(modName).isPresent())
+                if (!ModList.get().getModContainerById(modName).isPresent())
                 {
                     structure[y][z][x] = (short) pallete.indexOf(Blocks.AIR.getDefaultState());
                     continue;
                 }
```

Comment:
To clarify, I'm saying that this part of the mostly-existing code is pointless, but also harmless.  So it could be removed if people feel strongly enough about it but I was inclined to leave it for backwards consistency.  The rest of the discussion wasn't really related to any code in this PR.

Reply:
Well, this is blueprint creation, so indeed it would be weird trying to scan a black that's not present (as far as I understood).